packages/playwright-test/src/util.ts

```diff
@@ -278,3 +278,67 @@ export function getPackageJsonPath(folderPath: string): string {
   return result;
 }
 
+export const levenshteinDistance = (s: string, t: string) => {
+  // See https://en.wikipedia.org/wiki/Levenshtein_distance for reference.
+  // This is a DP implementation variant.
+  const n = s.length;
+  const m = t.length;
+
+  if (n === 0)
+    return m;
+  if (m === 0)
+    return n;
+
+  const d: number[][] = [];
+  // m rows, n cols
```

Comment:
nit: I believe the usual agreement for matrixes is that N is for first dimension, and M is for second. In other words, N for rows, M for columns.

Reply:
It might be locale (and/or discipline math v. cs) specific! m comes before n alphabetically, so I've mostly seen "we have an m x n matrix  with m rows and n cols).